Dockerfiles/mlpack-dependencies-debian/Dockerfile

```diff
@@ -0,0 +1,58 @@
+#Using debian's latest image as base-image
+FROM debian:latest
+
+RUN apt-get update  && apt-get install -y aptitude && apt-get purge -y \
+    $(aptitude search '~i!~M!~prequired!~pimportant!~R~prequired! \
+    ~R~R~prequired!~R~pimportant!~R~R~pimportant!busybox!grub!initramfs-tools' \
+    | awk '{print $2}') && apt-get purge -y aptitude && \
+    apt-get autoremove -y && apt-get clean && rm -rf /usr/share/man/?? && \
+    rm -rf /usr/share/man/??_*
+
+#Installing dependencies required to run mlpack
+RUN apt-get update  && apt-get install -y --no-install-recommends wget \
+    libboost-math-dev libboost-program-options-dev libboost-test-dev \
+    libboost-serialization-dev cmake g++ binutils-dev make g++ txt2man git \
+    doxygen liblapack-dev libblas-dev libarpack2 libsuperlu-dev  && \
```

Comment:
Technically wget isn't needed anymore, but I doubt that removing it will make any serious size difference.  Also you have `g++` listed twice. :)